Rancher_Embedded_Retail_v0.6.adoc

```diff
@@ -15,29 +15,40 @@
 
 SLE Micro is an ultra-reliable, lightweight operating system purpose built for containerized and virtualized workloads. It leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise and merges them with a modern, immutable, developer-friendly OS platform.
 
 SLE Micro's immutable design ensures the OS is not altered during runtime and runs reliably every single time. Further, it leverages enterprise-hardened SLE common code base to provide enterprise-grade quality and reliability.
 
-K3s is a complete, CNCF certified Kubernetes distribution packaged as a single < 40MB binary that reduces the dependencies and steps needed to install, run and auto-update a production Kubernetes cluster. K3s run on all x86_64 platforms but is also optimized for ARM64 and ARMv7.
+<!-- 40MB was the compressed binary. I don't think that's the case today. At runtime, the agent and server together use about 500MB -->
+K3s is a complete, CNCF certified Kubernetes distribution packaged as a single binary that reduces the dependencies and steps needed to install, run and auto-update a production Kubernetes cluster. K3s runs on all x86_64 and aarch64 platforms but is also optimized for ARMv7.
+
+Utilizing a standardized O/S plus Kubernetes image across all Point-of-Service units significantly reduces the maintenance required for the entire application stack. This design also leaves the uniqueness of each unit (i.e. point-of-purchase-lane-2 vs. point-of-purchase-lane-4 vs. general-manager-station) in the hands of the Kubernetes-native application layer. This means that in the dynamic environment of a retail location, all Point-of-Service units are fully interchangeable, highly secure, flexible, and easy to deploy, manage and update remotely or in-store.
 
-Utilizing a standardized O/S plus Kubernetes image across all Point-of-Service units significantly reduces the maintenance required for most of the software stack. This design also leaves the uniqueness of each unit (i.e. point-of-purchase-lane-2 vs. point-of-purchase-lane-4 vs. general-manager-station) in the hands of the Kubernetes-native application layer. This means that in the dynamic environment of a retail location, all Point-of-Service units are fully interchangeable, highly secure, flexible, and easy to manage.
+<!-- Fundamentally the architecture is a store unit that hosts and OS and container orchestration as a foundation to all new and existing cloud-native appliacations. Each in store system is configured to register itself with a system in the retail enterprise datacenter or cloud where it checks in for software updates and can be remotely managed giving visibillity in the live running systems in-store when needed. SUSE professional services can help build custom solutions around these basic building blocks to enable retail enterprises to evolve into modern systems like those used in the cloud and highly optimized data centers around the world. -->
```

Comment:
I probably didn't describe this well enough but the idea is to have a Rancher server in each store to support the elasticity of functionality across the homogenized PoS systems. I was concerned that having central, or even regional servers would introduce latencies in removing/installing applications. Since the in store staff will have no technical skills, having things change very predictably helps keep their confidence in the tasks they are charged with.